packages/template-retail-react-app/app/components/social-login/index.jsx

```diff
@@ -76,7 +77,9 @@ const SocialLogin = ({form, idps}) => {
                 redirectURI: redirectURI
             })
         } catch (error) {
-            const message = formatMessage(API_ERROR_MESSAGE)
+            const message = /redirect_uri doesn't match/.test(error.message)
+                ? formatMessage(FEATURE_UNAVAILABLE_ERROR_MESSAGE)
```

Comment:
To think about this message a little bit, if I recall correctly, if we the social config is not set up correctly, the button to login using socialLogin would not show. This implies the feature should be available. Should we use a better error message? 🤔

Reply:
why don't we use a generic error instead for simplicity?

Reply:
When discussing with Nathan we decided that if the user has not configured this feature correctly, then we should show this "feature not available" error. Even if the config is not set up correctly, if the user has enabled social login in the config, the button using social login will show.